cli/src/cli.mjs

```diff
@@ -138,6 +140,19 @@ if ( parsed.help ) {
 
   usage();
 
+} else if ( parsed.addUser ) {
+
+  const user = parsed.addUser;
+
+  msg(`adding user ${user}`);
+  axios.put(url(host, port, `/api/v1/user/${ encodeURIComponent(user)}`), undefined)
```

Comment:
Not sure if it's on a different branch, but assuming we have both POST and PUT support, I would assume this would test whether a user id value was given and do one or the other (assuming we only have one CLI command/param).